devel/haskell-stan/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                haskell-stan
+version             0.1.0.1
+revision            0
+categories          devel haskell
+maintainers         nomaintainer
+license             Mozilla Public License 2.0
+
+description         Haskell STatic ANalyser
+
+long_description    Stan is a Haskell STatic ANalysis tool.
+
+homepage            https://github.com/kowainik/stan
+
+set plain_name      stan
```

Comment:
no ness for a variable here since you use it only once; just set it directly on the line below.

Reply:
Thanks, fixed.